proposals/overload-resolution-priority.md

```diff
@@ -258,6 +258,32 @@ Our options are:
 
 We will go with 1.
 
+### Further application errors
+
+There are a few more locations like [this](#application-error-or-warning-on-override-answered) that need to be confirmed. They include:
+
+* Conversion operators - The spec never says that conversion operators go through overload resolution, so the implementation blocks application on these members.
+  Should that be confirmed?
+* Lambdas - Similarly, lambdas are never subject to overload resolution, so the implementation blocks them. Should that be confirmed?
+* Destructors - again, currently blocked.
+* Static constructors - again, currently blocked.
+* Local functions - These are not currently blocked, because they _do_ undergo overload resolution, you just can't overload them. This is simlar to how we don't
+  error when the attribute is applied to a member of a type that is not overloaded. Should this behavior be confirmed?
+
+### Langversion Behavior
+
+The implementation currently only issues langversion errors when `OverloadResolutionPriorityAttribute` is applied, _not_ when it actually influences anything. This
+decision was made because the there are APIs that the BCL will add (both now and over time) that will start using this attribute; if the user manually sets their
+language version back to C# 12 or prior, they may see these members and, depending our langversion behavior, either:
+
+* If we ignore the attribute in C# <13, run into an ambiguity error because the api is truly ambiguous without the attribute, or;
+* If we error when the attribute affected the outcome, run into an error that the api is unconsumable. This will be especially bad because `Debug.Assert(bool)`
+  is being de-prioritized in .NET 9, or;
+* If we silently change resolution, encounter potentially different behavior between machines if one machine understands the attribute and another doesn't.
```

Comment:
I guess we should have a disclaimer in the attribute docs that this can happen and that it should be used only on overloads which have equivalent behavior.